huaweicloud/services/acceptance/elb/resource_huaweicloud_elb_member_test.go

```diff
@@ -255,3 +289,137 @@ resource "huaweicloud_elb_member" "member_2" {
 }
 `, rName, rName, rName)
 }
+
+func testAccElbV3MemberConfig_crossVpcBackend_basic(rName string) string {
+	return fmt.Sprintf(`
+data "huaweicloud_vpc_subnet" "test" {
+  name = "subnet-default"
+}
+
+data "huaweicloud_availability_zones" "test" {}
+
+resource "huaweicloud_elb_loadbalancer" "test" {
+  name              = "%s"
+  cross_vpc_backend = true
+  ipv4_subnet_id    = data.huaweicloud_vpc_subnet.test.subnet_id
+  ipv6_network_id   = data.huaweicloud_vpc_subnet.test.id
+
+  availability_zone = [
+    data.huaweicloud_availability_zones.test.names[0]
+  ]
+}
+
+resource "huaweicloud_elb_listener" "test" {
+  name            = "%s"
+  description     = "test description"
+  protocol        = "HTTP"
+  protocol_port   = 8080
+  loadbalancer_id = huaweicloud_elb_loadbalancer.test.id
+
+  forward_eip = true
+
+  idle_timeout = 60
+  request_timeout = 60
+  response_timeout = 60
+}
+
+resource "huaweicloud_elb_pool" "test" {
+  name        = "%s"
+  protocol    = "HTTP"
+  lb_method   = "ROUND_ROBIN"
+  listener_id = huaweicloud_elb_listener.test.id
+}
+
+resource "huaweicloud_elb_member" "member_1" {
+  address       = "121.121.0.120"
+  protocol_port = 8080
+  pool_id       = huaweicloud_elb_pool.test.id
+
+  timeouts {
+    create = "5m"
+    update = "5m"
+    delete = "5m"
+  }
+}
+
+resource "huaweicloud_elb_member" "member_2" {
+  address       = "121.121.0.121"
+  protocol_port = 8080
+  pool_id       = huaweicloud_elb_pool.test.id
+
+  timeouts {
+    create = "5m"
+    update = "5m"
+    delete = "5m"
+  }
+}
+`, rName, rName, rName)
+}
+
+func testAccElbV3MemberConfig_crossVpcBackend_update(rName string) string {
+	return fmt.Sprintf(`
+data "huaweicloud_vpc_subnet" "test" {
+  name = "subnet-default"
+}
+
+data "huaweicloud_availability_zones" "test" {}
+
+resource "huaweicloud_elb_loadbalancer" "test" {
+  name              = "%s"
+  cross_vpc_backend = true
+  ipv4_subnet_id    = data.huaweicloud_vpc_subnet.test.subnet_id
+  ipv6_network_id   = data.huaweicloud_vpc_subnet.test.id
+
+  availability_zone = [
+    data.huaweicloud_availability_zones.test.names[0]
+  ]
+}
+
+resource "huaweicloud_elb_listener" "test" {
+  name            = "%s"
+  description     = "test description"
+  protocol        = "HTTP"
+  protocol_port   = 8080
+  loadbalancer_id = huaweicloud_elb_loadbalancer.test.id
+
+  forward_eip = true
+
+  idle_timeout = 60
+  request_timeout = 60
+  response_timeout = 60
+}
+
+resource "huaweicloud_elb_pool" "test" {
+  name        = "%s"
+  protocol    = "HTTP"
+  lb_method   = "ROUND_ROBIN"
+  listener_id = huaweicloud_elb_listener.test.id
+}
+
+resource "huaweicloud_elb_member" "member_1" {
+  address        = "121.121.0.120"
+  protocol_port  = 8080
+  weight         = 10
+  pool_id        = huaweicloud_elb_pool.test.id
+
+  timeouts {
```

Comment:
all of the `timeouts` block should be removed in the acceptance test

Reply:
done